modules/visual_script/doc_classes/VisualScriptSubCall.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptSubCall" inherits="VisualScriptNode" version="4.0">
 	<brief_description>
+		Calls a method called [code]_subcall[/code] in this object.
 	</brief_description>
 	<description>
+		[VisualScriptSubCall] will call method named [code]_subcall[/code] in the current script. It will fail if the method doesn't exist or the provided arguments are wrong.
```

Comment:
It wil call a method called _subcall in the ateached script.
It wil also display the name of the atached script if valid
PS: currently out of order

Reply:
The part about name of the script is unnecessary. It's just cosmetic thing.